scout/demo/643595.config.yaml

```diff
@@ -0,0 +1,61 @@
+---
+
+owner: cust000
+
+family: 'internal_id_2'
```

Comment:
Why did the case id change here?

Reply:
Since you want to test that the labels appear on *another* case?

Reply:
Yes, it is nice to have two test cases with the same variants to test behaviours like this one